### ISSUE TITLE
create term for 'increased inner canthal distance'

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#11493]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@azankl I will add as a synoynm for  [Telecanthus (HP:0000506)](https://hpo.jax.org/browse/term/HP:0000506)

**Comment by pnrobinson** [comment]
done
